src/imitation/algorithms/preference_comparisons.py

```diff
@@ -641,7 +641,8 @@ def __call__(
 
         # we need two fragments for each comparison
         for _ in range(2 * num_pairs):
-            traj = self.rng.choice(trajectories, p=np.array(weights) / sum(weights))
+            p = np.array(weights) / sum(weights)
+            traj = self.rng.choice(trajectories, p=p)  # type: ignore[arg-type]
```

Comment:
From https://github.com/HumanCompatibleAI/imitation/pull/594/files
```suggestion
            # numpy's typing thinks it can't accept a list here, but it can
            traj = self.rng.choice(trajectories, p=p)  # type: ignore[arg-type]
```